src/main/java/duke/ListCommand.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+public class ListCommand extends Command{
```

Comment:
```suggestion
public class ListCommand extends Command {
```
Missing whitespace between Command and {